frontend/src/utils/grid/cellRenderers.jsx

```diff
@@ -312,17 +312,18 @@ export const ReportsStatusRenderer = (props) => {
   const statusArr = [
     'Draft',
     'Submitted',
+    'Analyst adjustment',
     'Recommended by analyst',
     'Recommended by manager',
     'Assessed',
-    'Reassessed',
     'Rejected'
   ]
   const statusColorArr = [
```

Comment:
I'm not a fan of this index to index matching between these arrays. Having a direct mapping would make more sense.

Reply:
Converted both this and the one for transfers to use enums and a map.

Reply:
awesome thank you.